tests/conftest.py

```diff
@@ -1134,6 +1131,23 @@ def get_state_db(self) -> DVSDatabase:
 
         return self.state_db
 
+    def get_fvs_dict(self, fvs):
+        fvs_dict = {}
+        for fv in fvs:
+            fvs_dict.update({fv[0]:fv[1]})
+        return fvs_dict
+
+    def change_port_breakout_mode(self, intf_name, target_mode, options=""):
+        cmd = "config interface breakout %s %s -y %s"%(intf_name, target_mode, options)
+        self.runcmd(cmd)
+        time.sleep(2)
+
+    def verify_port_breakout_mode(self, intf_name, current_mode):
+        brkout_cfg_tbl = swsscommon.Table(self.cdb, "BREAKOUT_CFG")
+        (status, fvs) = brkout_cfg_tbl.get(intf_name)
+        assert(status == True)
+        fvs_dict = self.get_fvs_dict(fvs)
+        assert(fvs_dict["brkout_mode"] == current_mode)
```

Comment:
```suggestion
        self.get_config_db().wait_for_field_match(
            "BREAKOUT_CFG",
            intf_name,
            {"brkout_mode": current_mode}
        )
```

We can simplify + stabilize this by waiting for the brkout_mode to match the expected (or fail if it never does). I think this also removes the need for `get_fvs_dict` method (unless it's used somewhere else that I'm missing).